core/objectstore/metadata.go

```diff
@@ -19,8 +19,10 @@ const (
 
 // Metadata represents the metadata for an object.
 type Metadata struct {
-	// Hash is the hash of the object.
-	Hash string
+	// Hash256 is the 256 hash of the object.
+	Hash256 string
+	// Hash512_384 is the 512_384 hash of the object.
+	Hash512_384 string
```

Comment:
Just call this SHA-384 (SHA-512/... nomenclature is really only used for the 256 and 224 truncations of SHA-512)

Reply:
Fair, I did want to be explicit, but you are correct.